base/markdown/render/plain.jl

```diff
@@ -51,19 +51,19 @@ function plaininline(io::IO, md...)
 end
 
 plaininline(io::IO, md::Vector) = !isempty(md) && plaininline(io, md...)
-
-plaininline(io::IO, md::Image) = print(io, "![$(md.alt)]($(md.url))")
-
-plaininline(io::IO, s::String) = print(io, s)
-
-plaininline(io::IO, md::Bold) = plaininline(io, "**", md.text, "**")
-
-plaininline(io::IO, md::Italic) = plaininline(io, "*", md.text, "*")
-
+plaininline(io::IO, md::Image) = print(io, "![", plaininline(md.alt), "](",
+                                       md.url, ")")
+plaininline(io::IO, md::Link) = print(io, "[", plaininline(md.text), "](",
+                                      md.url, ")")
+plaininline(io::IO, md::Bold) = print(io, "**", plaininline(md.text), "**")
+plaininline(io::IO, md::Italic) = print(io, "*", plaininline(md.text), "*")
```

Comment:
Good to see the extra methods here but there are a lot of unnecessary stylistic changes – would be great to see the diff being as simple as possible